gradle/libs.versions.toml

```diff
@@ -1,12 +1,12 @@
 [versions]
 jdkTarget = "8"
 jvmToolchain = "21"
-kotlinTarget = "1.9.0"
-kotlinPlugin = "1.9.10"
-kotlinx-coroutines = "1.6.4"
-kotlinx-serialization = "1.5.1"
-ktor = "2.3.2"
-log4j = "2.20.0"
+kotlinTarget = "1.9.20"
```

Comment:
❌  Let's not bump the target version of Kotlin without an explicit reason.

I also updated the dependency versions in a separate PR (https://github.com/alllex/telegram-bot-kit/pull/2), so let's remove the version updates from this one